src/util/base-input.ts

```diff
@@ -229,6 +229,7 @@ export class BaseInput<T> extends Ion implements CommonInput<T> {
     this._form && this._form.unsetAsFocused(this);
     this._setFocus(false);
     this.ionBlur.emit(this);
+    this._onTouched && this._onTouched();
   }
```

Comment:
Thank you for submitting this PR.

It seems like this is only half of the fix. This will set ng-touched on blur, but it does not prevent touched from being applied on keyup (see #12102, #12700). I believe this same call also needs to be removed from `onChange()` on line 257.

I will need to ask around here about that, though.

Reply:
@kensodemann You're right.
If we look at Angular implementation this is the expected behavior.
I'll change that.

Reply:
@frontendbeast made a example about it http://plnkr.co/edit/8NtRId4YGbhXTHcWOMfV?p=preview

Reply:
After reviewing this with @manucorporat, this change is a little deeper than this. As we have it coded right now, it will have a negative impact on checkboxes and a couple of other inputs. We discussed solutions, which I will try to get around to implementing over the weekend (easier to do than try to explain).

Reply:
Thanks @kensodemann :+1: 
I'll appreciate this fix.